hawkular-accounts-feature-pack-dist/src/distribution/resources/modules/system/layers/base/org/jboss/as/product/hawkular-accounts/module.xml

```diff
@@ -0,0 +1,33 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+
+    Copyright 2015 Red Hat, Inc. and/or its affiliates
+    and other contributors as indicated by the @author tags.
+
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+       http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+
+-->
+<module xmlns="urn:jboss:module:1.1" name="org.jboss.as.product" slot="hawkular-accounts">
+
+  <properties>
+    <property name="jboss.api" value="private" />
+  </properties>
+
+  <resources>
+    <resource-root path="dir" />
+    <!-- Insert resources here -->
```

Comment:
I do not think so. I rather copied that blindly from somewhere. Let me remove that comment.